extra/utils/validation.js

```diff
@@ -0,0 +1,50 @@
+let users =
+    [
+        {
+            traineeEmail: "shubhasis1.padhi@successive.tech",
+            reviewerEmail: "shubham1.jain@successive.tech"
+        },
+        {
+            traineeEmail: "shubhasis2.padhi@successive.tech",
+            reviewerEmail: "shubham2.jain@successive.tech"
+        },
+        {
+            traineeEmail: "shubhasis13.padhi@successive.tech",
+            reviewerEmail: "shubham13.jain@successive.tech"
+        },
+        {
+            traineeEmail: "shubhasis14.padhi@successive.tech",
+            reviewerEmail: "shubham4@.jain@successive.tech"
+        }
+    ];
+let emailregex = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+let vldUser = "", inVldUser = "", count1 = 0, count2 = 0;
```

Comment:
Use proper variable names for example validUsers, invalidUsers etc

Reply:
Made all the changes